### PR TITLE
Some translations update

### PR DESCRIPTION
**Pull request by ercanersoy**

### CODE REVIEW

**Comment by HBelusca** [comment]
> Conflicting files
> dll/shellext/acppage/lang/tr-TR.rc 
Please fix!

**Comment by ercanersoy** [comment]
What error on this file?

**Comment by learn-more** [comment]
The resource string `IDS_TABTITLE` should now be `IDS_COMPAT_TITLE`.
And there are a lot of commits in between adding / removing translations again?

**Comment by ercanersoy** [comment]
I have updated my repository.

**Comment by sstopkin** [comment]
@ercanersoy Can you update your branch again, please?

dll/shellext/acppage/lang/tr-TR.rc (Added new LTEXT before CONTROL)
```
LTEXT "",IDC_ENABLED_LAYERS,7,185,210,21
CONTROL "<A>Program uyumluluğu</A> konusunda daha çok öğren.", IDC_INFOLINK, "SysLink", WS_CHILD | WS_TABSTOP | WS_VISIBLE, 7, 207, 210, 10, WS_EX_NOPARENTNOTIFY
```

dll/shellext/fontext/fontext.rc (Added RO translation)
```
#include <windef.h>

#include "resource.h"

#define REACTOS_VERSION_DLL
#define REACTOS_STR_FILE_DESCRIPTION  "ReactOS Font Folder"
#define REACTOS_STR_INTERNAL_NAME     "fontext"
#define REACTOS_STR_ORIGINAL_FILENAME "fontext.dll"
#include <reactos/version.rc>

/* UTF-8 */
#pragma code_page(65001)

#ifdef LANGUAGE_EN_US
    #include "lang/en-US.rc"
#endif
#ifdef LANGUAGE_RO_RO
    #include "lang/ro-RO.rc"
#endif
#ifdef LANGUAGE_TR_TR
    #include "lang/tr-TR.rc"
#endif```

**Comment by ercanersoy** [comment]
I have updated this pull request. But, I have make a mistake of naming commit.

File: dll/shellext/fontext/fontext.rc [hunk 1]
```diff
@@ -14,6 +14,9 @@
 #ifdef LANGUAGE_EN_US
     #include "lang/en-US.rc"
 #endif
+#ifdef LANGUAGE_RO_RO
+    #include "lang/ro-RO.rc"
+#endif
```
Comment (HBelusca) [inline dll/shellext/fontext/fontext.rc]: Why is that, whereas there is just no romanian resource language for fontext?

Comment (ercanersoy) [reply dll/shellext/fontext/fontext.rc]: This file is exist:
https://github.com/reactos/reactos/blob/master/dll/shellext/fontext/lang/ro-RO.rc

**Comment by ercanersoy** [comment]
Why the pull request doesn't accepted?

**Comment by binarymaster** [comment]
@eersoy93 do you see there are conflicts? Resolve them.

**Comment by ercanersoy** [comment]
What do I for this reason?

**Comment by ercanersoy** [comment]
I have closed pull request. I will fix it. I will create a new one.

**Pull request closed without merging**